flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-mysql/src/main/java/org/apache/flink/cdc/connectors/mysql/source/MySqlDataSourceOptions.java

```diff
@@ -313,4 +313,12 @@ public class MySqlDataSourceOptions {
                     .defaultValue(true)
                     .withDescription(
                             "Whether to use legacy json format. The default value is true, which means there is no whitespace before value and after comma in json format.");
+
+    @Experimental
+    public static final ConfigOption<Boolean> SCAN_INCREMENTAL_SNAPSHOT_ASSIGN_ENDING_CHUNK_FIRST =
+            ConfigOptions.key("scan.incremental.snapshot.assign-ending-chunk-first.enabled")
```

Comment:
How about ```suggestion
            ConfigOptions.key("scan.incremental.snapshot.boundary-chunk-first.enabled")
```? IIUC, this options is used to control the boundary chunks should be assigned first or not?

Reply:
IIUC, should it be `ConfigOptions.key("scan.incremental.snapshot.unboundary-chunk-first.enabled")`? The end is the largest and the start is the smallest unbounded chunks.

Reply:
I mean ConfigOptions.key("scan.incremental.snapshot.boundary-chunk-first.enabled") with default value `true`, currently we assign first chunk(unboundary chunk) and then assign boundary chunk, right?

But the two ConfigOptions make sense to me when you use different default value, feel free to choose you flavor.

Reply:
Given that "unboundary" is not a commonly used term and may cause confusion, I have decided to use "unbounded" instead. Therefore, the configuration item will be `scan.incremental.snapshot.unbounded-chunk-first.enabled`.

Reply:
+1 from my side, please keep go on